sql/explain.go

```diff
@@ -64,9 +78,24 @@ func markExplain(plan planNode, mode explainMode) (planNode, error) {
 		return t, nil
 
 	case *sortNode:
-		return markExplain(t.plan, mode)
+		return markDebug(t.plan, mode)
 
 	default:
 		return nil, fmt.Errorf("TODO(pmattis): unimplemented %T", plan)
 	}
 }
+
+func populateExplain(v *valuesNode, plan planNode, level int) {
+	var name, description string
+	children := plan.ExplainPlan(&name, &description)
+
+	row := make(parser.DTuple, len(v.columns))
```

Comment:
`parser.DTuple` has the type `[]parser.Datum`. I'm allocating a slice to a particular length, how can that be a literal?